### DISCUSSION TITLE
Difference between OSS and Pro Verions

### DISCUSSION THREAD

**Discussion by vijaydsk** [rustdesk/rustdesk#10351] [Q&A]
Hello all,
Can any body share a doc or link to understand the differences between Rustdesk OSS Version and PRO Version.
You help is highly appreciated.
Thanks in advance.

**Comment by rustdesk** [comment]
Go to [discord](https://discord.gg/nDceKgxnkV) please, search there first, some users might have asked similar questions. GitHub is a place to talking about code and bug.
